src/hapi/pipelines/app/__main__.py

```diff
@@ -86,6 +86,13 @@ def parse_args():
         action="store_true",
         help="Debug",
     )
+    parser.add_argument(
+        "-flg",
+        "--flag-to-hdx",
```

Comment:
This could be --err-to-hdx and maybe -ehx as flg is a bit too generic?

Reply:
Good idea!